actionpack/test/lib/test_component.rb

```diff
@@ -37,5 +37,5 @@ def self.compile
   end
 
 private
-  attr_reader :title, :view_context
+  attr_reader :view_context
 end
```

Comment:
Why do we include this weighty thing in here even? Could we replace with something closer to `Class.new { def render_in(view_context); "hey"; end }`?